jujugui/static/gui/src/app/app.js

```diff
@@ -515,15 +515,7 @@ YUI.add('juju-gui', function(Y) {
       // Store the initial model UUID.
       const modelUUID = this._getModelUUID();
       this.set('modelUUID', modelUUID);
-      let user = null;
-      let password = null;
-      let macaroons = null;
-      let credentials = this.user.controller;
-      if (credentials.areAvailable) {
-        user = credentials.user;
-        password = credentials.password;
-        macaroons = credentials.macaroons;
-      }
+      const controller = this.user.controller;
```

Comment:
That would be misleading. I'll switch it to controllerCreds, since that's what it is.